getting_started/topics/first-boot/distribution-files-product.adoc

```diff
@@ -2,18 +2,46 @@
 [id="standalone-install"]
 === Installing the {project_name} server
 
-For this sample instance of {project_name}, this procedure involves installation in standalone mode. The server download ZIP file contains the scripts and binaries to run the {project_name} server. You can install the server on Linux or Windows.
+For this sample instance of {project_name}, this procedure involves installation in standalone mode. The server download ZIP file contains the scripts and binaries to run the {project_name} server. You install the 7.4.0 server first, then the {project_version} server patch.
 
```

Comment:
@andymunro The last rewritten sentence: "You install the 7.4.0 server first, then the {project_version} server patch." does not make sense. It is getting started guide -> first boot.